runtimevar/runtimevar.go

```diff
@@ -266,15 +276,10 @@ func (c *Variable) Latest(ctx context.Context) (Snapshot, error) {
 // CheckHealth returns an error unless Latest will return a good value
 // without blocking.
 func (c *Variable) CheckHealth() error {
-	haveGood := false
-	select {
-	case <-c.haveGood:
-		haveGood = true
-	default:
-	}
+	haveLatest := c.haveLatest()
```

Comment:
Can you keep the variable/function named `haveGood` (here and above)? I think that's more accurate (and matches the channel) than `haveLatest`.

Reply:
I wanted to do that but *Variable holds a haveGood channel member so we can't add a method with the same name.

Reply:
How about renaming the `haveGood` channel member `haveGoodCh`?